src/marks/template.jsx

```diff
@@ -78,7 +78,7 @@ export function MarksElement({
       <style>
         {`
                     .${CLASS.TEXT} {
-                        font-family: "Helvetica Neue", Helvetica, Arial, sans-serif;
```

Comment:
did they say we should remove `Helvetica`?

Reply:
I used the `font-family` value from the [font css file](https://www.paypalobjects.com/paypal-ui/web/fonts-and-normalize/2-0-0/fonts-and-normalize.min.css)

Happy to add back in Helvetica after Helvetica Neue

Reply:
I'm thinking its there as a fallback since we don't load the fonts for the SDK. if we can't find the neue version, fallback to regular Helvetica which all Apple products should have. then arial, then finally just any sans-serif